src/url.js

```diff
@@ -134,4 +134,10 @@ const setBaseUrl = (baseUrl) => {
   BASE_URL = baseUrl;
 };
 
-export {getParamString, getUrlEncodedParam, getUrl, getBaseUrl, setBaseUrl};
+export {
```

Comment:
Should this just be the default export now that this file is called `url`?

Reply:
It could, yeah.